src/components/options/options.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
I think it is better to store this component inside the `pattern` folder, in pages, because it is used only for patterns. Is it right?

Reply:
this component is universal and may be used  in many places where data is shown in the table